pkg/statdb/statdb.go

```diff
@@ -26,6 +27,8 @@ type DB interface {
 	FindInvalidNodes(ctx context.Context, nodeIDs storj.NodeIDList, maxStats *NodeStats) (invalid storj.NodeIDList, err error)
 	// Update all parts of single storagenode's stats.
 	Update(ctx context.Context, request *UpdateRequest) (stats *NodeStats, err error)
+	// UpdateNodeStats takes a NodeStats and updates the applicable entries
+	UpdateOperator(ctx context.Context, node storj.NodeID, updatedStats *NodeStats) (stats *NodeStats, err error)
```

Comment:
If we are updating just the operator fields here, we should have `updatedOperator *pb.NodeOperator` param instead of `updatedStats *NodeStats`.

Reply:
This is more about the param type instead of just the name.